### ISSUE TITLE
Automate Granting "Service Presence Status Access" (Omni-Channel)

### ISSUE THREAD

**Issue by ruslan-kurchenko** [amtrack/sfdx-browserforce-plugin#564]
By default, Salesforce does not provide an ability to deploy access entries and in scratch, org users do not have access to Presence Statuses. The only one available is "Offline". It blocks running standard testing and running e2e tests as a real service agent by accepting cases, etc. 

Idea: https://ideas.salesforce.com/s/idea/a0B8W00000Gdq3UUAR/add-service-presence-statuses-access-to-metadata-api

As mentioned in the DX@Scale Slack thread, it is worth adding this automation to Browserforce - https://dxatscale.slack.com/archives/CN9N54GF5/p1684482930346419

## Upvote & Fund

- We're using [Polar.sh](https://polar.sh/amtrack) so you can upvote and help fund this issue.
- We receive the funding once the issue is completed & confirmed by you.
- Thank you in advance for helping prioritize & fund our backlog.

<a href="https://polar.sh/amtrack/sfdx-browserforce-plugin/issues/564">
<picture>
  <source media="(prefers-color-scheme: dark)" srcset="https://polar.sh/api/github/amtrack/sfdx-browserforce-plugin/issues/564/pledge.svg?darkmode=1">
  <img alt="Fund with Polar" src="https://polar.sh/api/github/amtrack/sfdx-browserforce-plugin/issues/564/pledge.svg">
</picture>
</a>

**Comment by amtrack** [comment]
@ruslan-kurchenko Sounds good!
I don't have much experience with Omni Channel. But from what you and others have described it really looks like a Metadata Gap and therefore it's a perfect fit for `sfdx-browserforce-plugin`.

Do you want to create a PR yourself?

If you don't want to create a PR, I can do it.
Your script looks good, so I don't think the implementation itself will be complicated.
The thing that will require some effort is to develop the E2E tests for it.

What would be a good test case and what are the steps to prepare a Scratch Org for testing?

**Comment by ruslan-kurchenko** [comment]
Yes, I will contribute. It might take a bit of time, I need to get better knowledge about the plugin, but I got the main concept.

**Comment by amtrack** [comment]
Wonderful! Let me know if you have any questions...

**Comment by amtrack** [comment]
@ruslan-kurchenko FYI: This was just implemented by @ClayChipps in #658
